Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -5636,6 +5636,10 @@ struct WebCore::ServiceWorkerImportedScript {
     String mimeType;
 };
 
+[CreateUsing=fromIPCData] class WebCore::ScriptBuffer {
+    WebCore::ScriptBuffer::IPCData ipcData();
```

Comment:
The metadata will need the definition of IPCData somewhere, too.  With similar things I've put std::variant here even though it has a more concise name.

Reply:
Can I just add the typedef in the serialization.in? 
Using IPCData = …;